dle/localdeploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+mv .env.example .env
+sed '/MEM_LIMIT.*/d' .env > .env.tmp
+mv .env.tmp .env
+sed '/PROVISION_ES.*/d' .env > .env.tmp
+mv .env.tmp .env
+
+# Uncomment these if database has never been created. May take several hours.
```

Comment:
`LOAD` will probably take several days. Ingesting into Elastic, more like 1-3h

Reply:
Yes, I commented out the LOAD. So, only should be used if someone is running from scratch.